python/main.py

```diff
@@ -21,26 +21,27 @@
     allow_methods=["GET","POST","PUT","DELETE"],
     allow_headers=["*"],
 )
-dbpath = pathlib.Path(__file__).parent.parent.resolve() / "db" / "mercari.sqlite3"
+sqlpath = pathlib.Path(__file__).parent.parent.resolve() / "db" / "mercari.sqlite3"
 db_setup_path = pathlib.Path(__file__).parent.parent.resolve() / "db" / "items.db"
```

Comment:
I'm sorry for miscommunication. What I want to say is kind of like below, but this is not mandatory :bow:

```suggestion
dbpath = pathlib.Path(__file__).parent.parent.resolve() / "db" / "mercari.db"
db_setup_path = pathlib.Path(__file__).parent.parent.resolve() / "db" / "items.sql"
```